marlowe-test/chain/upload.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+gsutil -m -h 'Cache-Control:public,max-age=600' rsync -x ".*\.csv$" out gs://testing.marlowe.run/database/
+
```

Comment:
What's this about?

Reply:
Uploading test results to https://testing.marlowe.run/database/. This is temporary until we have automated overnight runs.